src/main/kotlin/no/nav/familie/tilbake/kravgrunnlag/batch/HåndterGamleKravgrunnlagBatch.kt

```diff
@@ -50,17 +50,17 @@ class HåndterGamleKravgrunnlagBatch(
                     beregnBestemtDato(KONTANTSTØTTE),
                 )
 
-            val frakobletKravgrunnlagGruppertPåEksternFagsakId = frakobletKravgrunnlag.groupBy { it.eksternFagsakId }
+            val frakobletKravgrunnlagGruppertPåEksternFagsakIdOgYtelsetype = frakobletKravgrunnlag.groupBy { Pair(it.eksternFagsakId, it.ytelsestype) }
```

Comment:
Legger til `ytelsetype` i grupperingen for å kunne skille mellom barnetrygd og kontantstøtte fagsaker. Vi bruker `Long` som id i begge appene og overlapp kan forekomme.